checker/src/main/java/org/checkerframework/checker/determinism/qual/RequiresDetToString.java

```diff
@@ -0,0 +1,15 @@
+package org.checkerframework.checker.determinism.qual;
+
+import java.lang.annotation.*;
+import org.checkerframework.framework.qual.InheritedAnnotation;
+
+/**
+ * Writing this annotation on a method declaration will ensure that if it has any parameter of type
```

Comment:
This describes how the checker enforces this annotation, but not what it means.  A method with this annotation requires that all arguments corresponding to `Object` parameters must return a deterministic String when toString is called on them.